src/views/Scan.js

```diff
@@ -0,0 +1,88 @@
+/**
```

Comment:
I think it's easier if you delete this file, idk if we will have merge conflicts if you leave it like this

Reply:
I can reduce the stuff that's on here but would rather not delete it since Standford's nav makes a reference to it now.

Reply:
stanford actually uses a component named ScanScreen or smth, either way i think its fine to delete it.

Reply:
When it was merged, the two Scan.js file merged into the Scan.js file that I had on my branch. I was a bit confused then, but now I've got the idea and it should be fixed!!